appinventor/components/src/com/google/appinventor/components/runtime/ChartDataModel.java

```diff
@@ -57,15 +58,6 @@ public abstract class ChartDataModel<
   */
   protected int maximumTimeEntries = 200;
```

Comment:
Note that `entries` and `maximumTimeEntries` were moved up to DataModel so they should not be declared here.